plugins/action/generate_cloud_modules.py

```diff
@@ -1325,6 +1325,22 @@ def generate_vmware_rest(args: Iterable, role_path: str):
                     role_path=role_path
                 )
                 module_list.append(module.name)
+
+    meta_dir = pathlib.Path(args.get("target_dir") + "/meta")
+    meta_dir.mkdir(parents=True, exist_ok=True)
+
+    yaml_dict = {
+        "requires_ansible": """>=2.14.0""",
```

Comment:
I prefer passing this as an argument instead of embedding the value directly. However, I'm uncertain about the method and source for passing it. One immediate idea is to use it as an argument in the manifest file. We could create a separate PR to include the Ansible version in the manifest file. If you have a better solution in mind, kindly share it.

Reply:
I agree, having this hard-coded is not very great. I also think that the manifest file would be a good (probably the best) place to put the ansible-core requirement.

I'll try to figure out how to do it and open a new PR. In the meantime, I'll mark this one as work in progress.